Assets/Script/Gameplay/Visuals/StrikelinePositioner.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections;
+using DG.Tweening;
+using UnityEngine;
+using YARG.Gameplay.HUD;
+using YARG.Gameplay.Player;
+using YARG.Settings.Preview;
+
+namespace YARG.Gameplay.Visuals
+{
+    public class StrikelinePositioner : MonoBehaviour
+    {
+        private const float GLOBAL_ANIM_DELAY = 2f;
+        private const float LOCAL_ANIM_OFFSET = 0.1f;
+
+        private const float ANIM_FRET_ZOOM_DELAY         = 0.5f;
+        private const float ANIM_BASE_TO_PEAK_INTERVAL   = 0.167f;
+        private const float ANIM_PEAK_TO_VALLEY_INTERVAL = 0.083f;
+        private const float ANIM_INIT_Z_OFFSET           = -1.2f;
+        private const float ANIM_PEAK_Z_OFFSET           = 0.1f;
+
+        private float     _zOffset;
+        private Coroutine _coroutine;
+
+        private void Start()
+        {
+            _zOffset = transform.localPosition.z;
+            _coroutine = StartCoroutine(RaiseStrikeline(true));

```

Comment:
Here as well, with the same caveat about this probably not being the best way of doing it, just a way that works.

```suggestion
            if (FindObjectOfType<GameManager>() != null)
            {
                _coroutine = StartCoroutine(RaiseStrikeline(true));
            }
```